test/PosthogAnalytics-test.ts

```diff
@@ -0,0 +1,194 @@
+import { Anonymity, getRedactedCurrentLocation, IAnonymousEvent, IPseudonymousEvent, IRoomEvent,
+    PosthogAnalytics } from '../src/PosthogAnalytics';
```

Comment:
```suggestion
import {
    Anonymity,
    getRedactedCurrentLocation,
    IAnonymousEvent,
    IPseudonymousEvent,
    IRoomEvent,
    PosthogAnalytics,
} from '../src/PosthogAnalytics';
```

Reply:
I guess ideally the linter would autoformat to this, if that's what we want?

Reply:
+1 for better linter rules